apps/src/blockly/addons/cdoUtils.js

```diff
@@ -168,6 +168,14 @@ export function handleColorAndStyle(block, color, style) {
 }
 
 export function setHSV(block, h, s, v) {
+  if (block.getStyleName()) {
+    console.warn(
```

Comment:
It looks like maybe this needs staging merged into it now?

Reply:
Oh good catch! Will fix tomorrow.